src/Server.cpp

```diff
@@ -179,6 +179,18 @@ void Server::removeChannel(const std::string &ch_name) {
 	std::cout << "remove Channel: " << ch_name << std::endl;
 }
 
+void Server::removeUser(const int fd) { this->user_map_.erase(fd); }
+
+void Server::removePollfd(const int fd) {
+	size_t i = 0;
+
+	for (std::vector<struct pollfd>::iterator it = pollfd_vec_.begin();
+		 it != pollfd_vec_.end(); ++it) {
+		if (it->fd == fd)
+			this->pollfd_vec_.erase(pollfd_vec_.begin() + i);
+	}
```

Comment:
```suggestion
	for (size_t i = 0; i < this->pollfd_vec_.size(); ++i) {
		if (this->pollfd_vec_.at(i).fd == fd) {
			this->pollfd_vec_.erase(pollfd_vec_.begin() + i);
		}
	}
```
イテレーターを使ってeraseしたコードだと、なぜかセグフォしてしまったので、上記のコードに変更して下さい